partiql-tests-runner/README.md

```diff
@@ -0,0 +1,12 @@
+# PartiQL Kotlin Test Runner
```

Comment:
This Gradle module is nearly the same as the one from `partiql-tests` with only the following changes:
- Changing `build.gradle.kts` to `build.gradle`
- Including `partiql-tests-runner` in the root project's `settings.gradle` file
- Add git submodule pointing to `partiql-tests`
- Changing paths to `partiql-tests` for reading the test data
- Some renaming of the package directories